src/app/(main)/[slug]/page.tsx

```diff
@@ -45,16 +47,21 @@ async function Page({ params }: Props) {
   const { slug } = params;
   const { perspective, isDraftMode } = getDraftModeInfo();
 
-  const [initialPage, initialBlogPage, initialCompensationsPage] =
-    await Promise.all([
-      loadQuery<PageBuilder>(SLUG_QUERY, { slug }, { perspective }),
-      loadQuery<BlogPage>(BLOG_PAGE_QUERY, { slug }, { perspective }),
-      loadQuery<CompensationsPage>(
-        COMPENSATIONS_PAGE_QUERY,
-        { slug },
-        { perspective },
-      ),
-    ]);
+  const [
+    initialPage,
+    initialBlogPage,
+    initialCompensationsPage,
+    initialLocationsData,
+  ] = await Promise.all([
+    loadQuery<PageBuilder>(SLUG_QUERY, { slug }, { perspective }),
+    loadQuery<BlogPage>(BLOG_PAGE_QUERY, { slug }, { perspective }),
+    loadQuery<CompensationsPage>(
+      COMPENSATIONS_PAGE_QUERY,
+      { slug },
+      { perspective },
+    ),
+    loadQuery<CompanyLocation>(COMPANY_LOCATIONS_QUERY, {}, { perspective }),
```

Comment:
Query returns a list, not just a single location, right?
```suggestion
    loadQuery<CompanyLocation[]>(COMPANY_LOCATIONS_QUERY, {}, { perspective }),
```